applications/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -466,20 +468,46 @@ static void scan_init(void)
 
 static void set_conn_params(struct bt_conn *conn, bool peer_llpm_support)
 {
-	struct bt_le_conn_param param = {
-		.latency = 99,
-		.timeout = 400
-	};
+	int err;
 
 	if (peer_llpm_support && IS_ENABLED(CONFIG_BT_LL_NRFXLIB)) {
-		param.interval_min = 0x0D01;
-		param.interval_max = 0x0D01;
+		struct net_buf *buf;
+
+		hci_vs_cmd_conn_update_t *cmd_conn_update;
+
+		buf = bt_hci_cmd_create(HCI_VS_OPCODE_CMD_CONN_UPDATE,
+					sizeof(*cmd_conn_update));
+		if (!buf) {
+			printk("Could not allocate command buffer\n");
+			return;
+		}
+
+		u16_t conn_handle;
+
+		err = bt_hci_get_conn_handle(conn, &conn_handle);
+		if (err) {
+			printk("Failed obtaining conn_handle (err %d)\n", err);
```

Comment:
we don't use printk - only logger

Reply:
Will fix. Thank you.

Reply:
No need. I already created a PR :)

Reply:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1630